metadata-ingestion/source_docs/feast_repository.md

```diff
@@ -0,0 +1,52 @@
+# Feast Repository
+
+For context on getting started with ingestion, check out our [metadata ingestion guide](../README.md).
+
+## Setup
+
+To install this plugin, run `pip install 'acryl-datahub[feast-repository]'`.
+
+## Capabilities
+
+This plugin extracts:
+
+- Entities as [`MLPrimaryKey`](https://datahubproject.io/docs/graphql/objects#mlprimarykey)
+- Features as [`MLFeature`](https://datahubproject.io/docs/graphql/objects#mlfeature)
+- Feature views and on-demand feature views as [`MLFeatureTable`](https://datahubproject.io/docs/graphql/objects#mlfeaturetable)
+- Batch and stream source details as [`Dataset`](https://datahubproject.io/docs/graphql/objects#dataset)
+
+## Quickstart recipe
+
+Check out the following recipe to get started with ingestion! See [below](#config-details) for full configuration options.
+
+For general pointers on writing and running a recipe, see our [main recipe guide](../README.md#recipes).
+
+```yaml
+source:
+  type: "feast-repository"
+  config:
+    # Coordinates
+    path: "/path/to/repository/"
+    # Options
+    environment: "PROD"
+
+sink:
+  # sink configs
+```
+
+## Config details
+
+Note that a `.` is used to denote nested fields in the YAML recipe.
+
+| Field         | Required | Default | Description                                |
+| ------------- | -------- | ------- | ------------------------------------------ |
+| `path`        | `yes`    |         | Path to Feast repository.                  |
```

Comment:
nit: for consistency, could you use checkmark icons for "Required"? ex: https://datahubproject.io/docs/metadata-ingestion/source_docs/postgres/#config-details

Reply:
Done! We lost alignment in raw, but that's ok!
![image](https://user-images.githubusercontent.com/10113621/154698972-137448ca-f33c-4688-8c0a-6dd68cb0c525.png)